src/hooks/useProposal/util.ts

```diff
@@ -72,8 +72,12 @@ export const formatAccountsByChain = (
 
   const mappedAccountsByChains: AccountsInChain[] = chainsDeduplicated.map(
     (chain) => {
-      const formatedChain = getCurrencyByChainId(chain);
+      let formatedChain = getCurrencyByChainId(chain);
 
+      if (formatedChain.startsWith("xrpl")) {
+        formatedChain = "ripple";
+      }
+      
```

Comment:
you are not suppose to do this, instead you should use the correct key in `RIPPLE_CHAINS`